machine_aarch64_test.go

```diff
@@ -0,0 +1,14 @@
+// +build aarch64 arm64
```

Comment:
Shouldn't we drop this and move this test into machine_test.go ?  There's nothing really arch specific here, only that at runtime qemu-system-$arch may or maynot support this machine type.